traffic_portal/app/src/modules/private/dashboard/dashboard.tpl.html

```diff
@@ -21,6 +21,11 @@
     <div ng-if="hasCapability('stats-read') && hasCapability('cdns-read')" class="row tile_count">
         <div ui-view="dashboardStatsContent"></div>
     </div>
+    <div class="row">
+        <div class="col-md-12 col-sm-12 col-xs-12">
+            <div ng-if="hasCapability('delivery-services')" class="x_panel" ui-view="deliveryServicesContent"></div>
```

Comment:
this isn't the proper capability. check the capability table for the list of capabilities.